Test/VSComp2010/Problem2-Invert.dfy

```diff
@@ -26,57 +25,44 @@
 // connect the two.
 
 method M(N: int, A: array<int>, B: array<int>)
```

Comment:
Did the contents of this test need to change to allow `// RUN: %testDafnyForEachResolver "%s"` to be used? Why is that?

Reply:
Sorry, maybe my question wasn't specific enough. My question was: why did you need the changes on lines 29-81? I see that the change on line 1 triggers the new resolver to be used, but why can't the rest of the test stay the same?

Reply:
The rest could have stayed the same. But I took the opportunity to remove deprecated semi-colons, use a `for` loop instead `while`, use the display initialization of a new array, and remove some assert that wasn't needed for the proof.